config/routes.rb

```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+
+  resources :users do
+    resources :posts
```

Comment:
- [ ] As requested in the project requirements, kindly ensure that routes maintain a RESTful URL, i.e. unnecessary endpoints are not exposed. presently the project is exposing more than the needed URL
![Screenshot 2022-09-20 182944](https://user-images.githubusercontent.com/85095702/191325930-5cf51e21-1730-4a6f-9801-f9ff8a875e2c.png)